edx_proctoring/api.py

```diff
@@ -881,38 +882,31 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
                 cascade_effects=False
             )
 
-    # email will be send when the exam is proctored and not practice exam
-    # and the status is verified, submitted or rejected
-    should_send_status_email = (
-        exam_attempt_obj.taking_as_proctored and
```

Comment:
The reason I removed it was due to code coverage. I needed to determine serve different templates for different status codes, so this check had to be done in the `create_proctoring_attempt_status_email` function. If the code also did the check beforehand, then the code could never go through the else clause. It seems cleaner to just let the function itself decide whether an email is needed.